### PR TITLE
Made basic UI for docs page

### PR DESCRIPTION
**Pull request by ghost**
Also formatted files with prettier

### CODE REVIEW

File: site/src/routes/docs/index.svelte [hunk 1]
```diff
@@ -1 +1,149 @@
...
+          class="w-4 h-4 ml-1"
+          viewBox="0 0 24 24"
+        >
+          <path d="M5 12h14M12 5l7 7-7 7" />
+        </svg>
+      </a>
+    </div>
+  </div>
+</header>
+<section class="bg-gradient-to-r from-gray-900 to-gray-800 ">
```
Comment (dhzdhd) [inline site/src/routes/docs/index.svelte:33]: This section seems unnecessary.
Also add a main tag with all sections enclosed within it.

File: site/src/routes/docs/index.svelte [hunk 2]
```diff
@@ -1 +1,149 @@
-<div>In development</div>
+<script lang="ts">
```
Comment (dhzdhd) [inline site/src/routes/docs/index.svelte:1]: This script tag is currently not needed.

**Review by dhzdhd — COMMENTED** [review]
Thanks for making the base blueprint for the documentation route!

The page is well done but needs a few changes.

- Format the entire file with the headwind tailwind formatter or any such formatter.
- The UI is not consistent with the landing page, try to fix that.

File: site/src/routes/docs/index.svelte (hunk 1 above)
Comment (Akulbcodes) [reply site/src/routes/docs/index.svelte:33]: image
So should I remove this whole thing?

Comment (dhzdhd) [reply site/src/routes/docs/index.svelte:33]: Remove the Documentation heading and Explore our guides... section

**Comment by Akulbcodes** [comment]
> Thanks for making the base blueprint for the documentation route!
> 
> The page is well done but needs a few changes.
> 
>     * Format the entire file with the headwind tailwind formatter or any such formatter.
> 
>     * The UI is not consistent with the landing page, try to fix that.

Headwind only works if a tailwind.config.js file is present

**Comment by dhzdhd** [comment]
> Headwind only works if a tailwind.config.js file is present
I think you can make a dummy tailwind config file for using headwind or find another formatter that does the job.

**Comment by Akulbcodes** [comment]
I can use the tailwind prettier plugin

**Comment by dhzdhd** [comment]
> I can use the tailwind prettier plugin

Yeah I think that would be the best way of solving this issue.

**Comment by Akulbcodes** [comment]
image
I am not very clear on what to do with this

File: site/src/routes/docs/index.svelte [hunk 3]
```diff
@@ -1 +1,149 @@
...
+<section class="bg-gradient-to-r from-gray-900 to-gray-800 ">
+  <h1
+    class="text-4xl font-bold text-green-500 pt-4 text-center md:text-left md:px-5 md:pt-12 md:pb-2"
+  >
+    Documentation
+  </h1>
+  <p class="p-0 text-xl text-gray-400 text-center md:text-left md:px-5">
+    Expore our guides to work with Obsidian
+  </p>
+</section>
```
Comment (dhzdhd) [inline site/src/routes/docs/index.svelte:42]: This is the part that is unnecessary.

**Comment by Akulbcodes** [comment]
image
I feel like this can be centered to match the website layout.

**Comment by dhzdhd** [comment]
> image I feel like this can be centered to match the website homepage.

Sure, do it

**Comment by dhzdhd** [comment]
The Obsidian documentation top heading can be smaller to be more consistent with the main page

**Comment by Akulbcodes** [comment]
> The Obsidian documentation top heading can be smaller to be more consistent with the main page

done

**Comment by Akulbcodes** [comment]
Wait is this the heading you meant

**Comment by dhzdhd** [comment]
> Wait is this the heading you meant

Yes

**Comment by dhzdhd** [comment]
> any more changes?

Increase the header height to be more consistent with the main page

**Comment by Akulbcodes** [comment]
> > any more changes?
> 
> Increase the header height to be more consistent with the main page

It's done I think

**Comment by dhzdhd** [comment]
> > > any more changes?
> > 
> > 
> > Increase the header height to be more consistent with the main page
> 
> It's done now I think

Are the latest changes formatted with prettier? If so, the code is ready to be merged

**Comment by Akulbcodes** [comment]
> > > > any more changes?
> > > 
> > > 
> > > Increase the header height to be more consistent with the main page
> > 
> > 
> > It's done now I think
> 
> Are the latest changes formatted with prettier? If so, the code is ready to be merged

Yes, there really were no files to format since I just added one utility class

**Comment by dhzdhd** [comment]
Alright then, LGTM

**Comment by dhzdhd** [comment]
LGTM

**Review by dhzdhd — APPROVED** [review]

**Comment by Akulbcodes** [comment]
Thanks, very cool

**Pull request merged**